tests/integration/states/file.py

```diff
@@ -290,9 +340,16 @@ def test_managed_file_with_grains_data(self):
         Test to ensure we can render grains data into a managed
         file.
         '''
-        ret = self.run_function('state.sls', ['file-grainget'])
-        self.assertTrue(os.path.exists('/tmp/file-grain-test'))
-        file_contents = open('/tmp/file-grain-test', 'r').readlines()
+        if IS_WINDOWS:
```

Comment:
The problem is that there's a corresponding state file with a path. How do you use `integration.TMP` or `tempfile.gettempdir()` in a state file?